arm/Microsoft.DataProtection/backupVaults/deploy.bicep

```diff
@@ -0,0 +1,127 @@
+@description('Required. Name of the Backup Vault.')
+param name string
+
+@description('Optional. Enable telemetry via the Customer Usage Attribution ID (GUID).')
+param enableDefaultTelemetry bool = true
+
+@description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Array of role assignment objects that contain the \'roleDefinitionIdOrName\' and \'principalId\' to define RBAC role assignments on this resource. In the roleDefinitionIdOrName attribute, you can provide either the display name of the role definition, or its fully qualified ID in the following format: \'/providers/Microsoft.Authorization/roleDefinitions/c2f4ef07-c644-48eb-af81-4b1b4947fb11\'.')
+param roleAssignments array = []
+
+@allowed([
+  'CanNotDelete'
+  'NotSpecified'
+  'ReadOnly'
+])
+@description('Optional. Specify the type of lock.')
+param lock string = 'NotSpecified'
+
+@description('Optional. Enables system assigned managed identity on the resource.')
+param systemAssignedIdentity bool = false
+
+@description('Optional. The ID(s) to assign to the resource.')
+param userAssignedIdentities object = {}
+
+@description('Optional. Tags of the Recovery Service Vault resource.')
+param tags object = {}
+
+@description('Optional. The datastore type to use.')
+@allowed([
+  'ArchiveStore'
+  'SnapshotStore'
+  'VaultStore'
+])
+param dataStoreType string = 'SnapshotStore'
+
+@description('Optional. The vault redundancy level to use.')
+@allowed([
+  'LocallyRedundant'
+  'GeoRedundant'
+])
+param type string = 'LocallyRedundant'
+
+@description('Optional. List of all backup policies.')
+param backupPolicies array = []
+
+var identityType = systemAssignedIdentity ? (!empty(userAssignedIdentities) ? 'SystemAssigned,UserAssigned' : 'SystemAssigned') : (!empty(userAssignedIdentities) ? 'UserAssigned' : 'None')
+
+var identity = identityType != 'None' ? {
+  type: identityType
+  userAssignedIdentities: !empty(userAssignedIdentities) ? userAssignedIdentities : null
+} : null
+
+var enableChildTelemetry = false
+
+resource defaultTelemetry 'Microsoft.Resources/deployments@2021-04-01' = if (enableDefaultTelemetry) {
+  name: 'pid-47ed15a6-730a-4827-bcb4-0fd963ffbd82-${uniqueString(deployment().name, location)}'
+  properties: {
+    mode: 'Incremental'
+    template: {
+      '$schema': 'https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#'
+      contentVersion: '1.0.0.0'
+      resources: []
+    }
+  }
+}
+
+resource bv 'Microsoft.DataProtection/backupVaults@2022-03-01' = {
```

Comment:
For consistency purposes we should proably use `backupVault` as a resource identifier

Reply:
Yes. I do agree that is a better name. Only named it ‘bv’ because the recoveryservicesvaults are named ‘rsv’.